stories/types.stories.tsx

```diff
@@ -24,6 +25,34 @@ const props = {
   };
 
 storiesOf('Types', module)
```

Comment:
How should I change my reviewing approach when looking at a storybook test versus a standard test? Is it just a render test? Could you explain better what I'm looking at?

Reply:
I'd say much lower standards, since it's _not_ a test. It's like... documentation? This one here is just a development helper. It makes it easier to do things locally.